Makefile

```
@@ -350,7 +350,7 @@ configure-v3:
 		sed -i "s/^send_rate = .*/send_rate = $(SEND_RECV_RATE)/" $(CONFIG_FILE); \
 		sed -i "s/^ttl-num-blocks = .*/ttl-num-blocks = 12/" $(CONFIG_FILE); \
 		sed -i "s/^ttl-duration = .*/ttl-duration = \"1m15s\"/" $(CONFIG_FILE); \
-		sed -i "s/^max_txs_bytes = .*/max_tx_bytes = 7897088/" $(CONFIG_FILE); \
+		sed -i "s/^max_tx_bytes = .*/max_tx_bytes = 7897088/" $(CONFIG_FILE); \
```

actually, needs to be the other way around:

```suggestion
		sed -i "s/^max_txs_bytes = .*/max_txs_bytes = 7897088/" $(CONFIG_FILE); \
```